key.ui/src/main/java/de/uka/ilkd/key/gui/settings/SettingsManager.java

```diff
@@ -120,7 +120,6 @@ private SettingsDialog createDialog(MainWindow mainWindow) {
         dialog.setSettingsProvider(settingsProviders);
         dialog.setDefaultCloseOperation(WindowConstants.DISPOSE_ON_CLOSE);
         dialog.setIconImage(IconFactory.keyLogo());
-        dialog.setSize(800, 600);
         dialog.setLocationByPlatform(true);
```

Comment:
```suggestion
```
I think we should delete this. It can lead to non-centered dialog (occurred on my windows machine), which might even be on another monitor than the main window.

Reply:
I think it would be best to remove all setLocationByPlatform calls at once in another PR since that will require more testing.

Reply:
Ok, let's move this to a separate PR, as it also affects some other dialogs.